newsletter-april-2021.rst

```diff
@@ -0,0 +1,66 @@
+.. post:: April 5, 2021
+   :tags: newsletter
+   :author: Juan Luis
+   :location: Madrid
+
+.. meta::
+   :description lang=en:
+      xxx
+
+Read the Docs newsletter - April 2021
+=====================================
+
+xxx
+
+Company highlights
+------------------
+
+-  We have a new colleague! Juan Luis will be working with us as
+   Developer Advocate to …
+-  xxx we saw an increment of X % page views
+-  We have `a new status page <http://status.readthedocs.com/>`__ so you
+   can have a glance at how our service is working.
+
+New features
+------------
+
+-  There is a `new support form <https://readthedocs.org/support/>`__ in
```

Comment:
I'd like this to the docs page, so it works for both .org/.com

Reply:
I just saw that there's also https://readthedocs.com/support/ for logged in users, but not sure what are you suggesting here?

Reply:
Linking to https://docs.readthedocs.io/en/latest/support.html -- which links out to both sites.

Reply:
Oh, right 👍🏽 Done!